templates/robotics/what-is-ros.html

```diff
@@ -87,7 +87,7 @@ <h2>What is ROS?</h2>
       }}
       <h2>What is ROS 2?</h2>
       <p>ROS 2 is a complete re-design of the framework, tackling the shortcomings of the first generation, effectively bringing it to industry needs and standards. ROS 2 contains new sets of packages that can be installed alongside ROS 1 to ease migration to a more secure platform. ROS 2 takes advantage of new technologies and newly updated APIs asked for by the community.</p>
-      <p><a class="p-button--positive" href="https://index.ros.org/doc/ros2/Tutorials/">Get started with ROS 2</a></p>
+      <p><a class="p-button--positive" href="https://docs.ros.org/en/humble/Tutorials.html">Get started with ROS 2</a></p>
```

Comment:
The link seems to be https://wiki.ros.org/ROS/Tutorials in the copy doc. Was there a change that I missed?